### DISCUSSION TITLE
Problem with states

### DISCUSSION THREAD

**Discussion by bobimicroweber** [livewire/livewire#4892] [1. Help]
Hello, im trying to update variable in custom method in my livewire component and i recive this error in browser.
Example:
$this->myarray['test']=1;

First time when i click on target method evrything is okay, but when i click second time i get this error in browser console

Uncaught TypeError: method is null
    value wire-directives.js:86
    get wire-directives.js:58
    debouncedHandler node_initializer.js:209
    value index.js:582
    debouncedHandler node_initializer.js:201
[wire-directives.js:86:37](https://microweber:8890/js/util/wire-directives.js)

Uncaught TypeError: name is null
    setAttribute index.js:151
    startLoading LoadingStates.js:222
    doAndSetCallbackOnElToUndo LoadingStates.js:281
    startLoading LoadingStates.js:220
    startLoading LoadingStates.js:208
    setLoading LoadingStates.js:179
    LoadingStates LoadingStates.js:68
    value MessageBus.js:17
    value MessageBus.js:16
    call HookManager.js:38
    callHook Store.js:121
    sendMessage index.js:244
    value index.js:252
    later debounce.js:8

**Comment by tushargugnani** [comment]
@bobimicroweber Can you share your component code.
